boards/stm32f769_discovery/src/sdcard.adb

```diff
@@ -223,8 +232,13 @@ package body SDCard is
          return;
       end if;
 
+      Enable_Clock (SDMMC_Controller (Controller.Device.all));
+      Reset (SDMMC_Controller (Controller.Device.all));
+
       Ret := STM32.SDMMC.Initialize
-        (Controller.Device.all, Controller.Info);
+        (Controller.Device.all,
+         System.BB.Board_Parameters.Main_Clock_Frequency,
```

Comment:
You should use STM32.Device.System_Clock_Frequencies.SYSCLK. That's what we use all across the library and it will remove the run-time dependency with the warnings off pragma.